common/lib/common-utils/package.json

```diff
@@ -68,9 +68,9 @@
     "@fluidframework/common-definitions": "^0.19.1",
     "@types/events": "^3.0.0",
     "assert": "^2.0.0",
-    "base64-js": "^1.3.1",
+    "base64-js": "^1.5.1",
     "events": "^3.1.0",
-    "lodash": "^4.17.19",
+    "lodash": "^4.17.20",
```

Comment:
Is this or the package lock change necessary?

Reply:
I think so, but I may need to go back on it when I update versions?  When I tried building with local symlinks, the webpack step was saying there were duplicate versions of these libraries; bumping the versions fixed this.

Reply:
I guess I'm confused. Why do you need to change the version of lodash for this change? I don't see it used at all

Reply:
I'm going to check this again and get back to you to make sure I didn't miss something.  (But basically all the pain you ran into when you tried to change server packages before applies to this scenario as well...)

Reply:
Ah that makes sense.

Reply:
ok yeah looks like it's unnecessary and it's a side effect of the local symlinking...  `common-utils` is already pulling in `4.17.20` during install for some reason while client is pulling in `4.17.19`.  it looks like client will pull in the updated version as well when updating the `common-utils` dependency after publishing, but that doesn't happen locally because we need to install in client before symlinking, and can't re-install after symlinking. bleh.